R/reorder.R

```diff
@@ -45,14 +51,31 @@
 #'   geom_point() +
 #'   geom_line() +
 #'   labs(colour = "Chick")
-fct_reorder <- function(.f, .x, .fun = median, ..., .desc = FALSE) {
+fct_reorder <- function(.f, .x, .fun = median, ..., .na_rm = NULL, .default = Inf, .desc = FALSE) {
```

Comment:
Want to do any `check_bool(.na_rm, allow_null = TRUE)` style checks?

Seems reasonable to not check `.default`

Reply:
Yeah, but I need to pull in the rlang helpers, so I'll do that in another PR.